content/realtime/channels.textile

```diff
@@ -255,6 +255,26 @@ channel.publish("action", data: "boom!")
 
 Normally, errors in attaching to a channel are communicated through the attach callback. For implicit attaches (and other cases where a channel is attached or reattached automatically, e.g. following the library reconnecting after a period in the @suspended@ state), there is no callback, so if you want to know what happens, you'll need to listen for channel state changes.
 
+<div lang="jsall,ruby,swift">
+
+h3(#transient-publish). Transient publishing
+
+When attempting to only publish to a client, it is possible to publish without attaching to the channel. This can be beneficial if you intend to publish into many channels, removing the need to attach to a channel each time you wish to publish.
```

Comment:
I think we're missing a key aspect of transient publishing being that by not attaching, the client is not subscribing to messages too, so this avoids messages being sent to the client. See https://docs.google.com/document/d/1CSJ3L6Mi2lkw_C26cG_QtdSwcdV8SaiQfEY5MBuENEc/edit#.

Reply:
I am a little concerned that if we do the docs like this, how will we know when to update this as those client SDKs add support?  Perhaps if we go down this route, we should at least add issues in those unsupported repos to update the docs once these features are fully supported. WDYT?